recipe/meta.yaml

```diff
@@ -12,6 +12,7 @@ source:
     - unreleased_changes.patch
     - unreleased_changes2.patch
     - fix_python_interface.patch
+    - fix_distutils.patch
 
 build:
   number: 2
```

Comment:
Perhaps we should bump this? The re-rendering brought a few changes.

Reply:
Good catch!